### ISSUE TITLE
Major changes 2023-10-09

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-releases#53]
MODs:
* NCBITaxon:4897\|Schizosaccharomyces japonicus | -8471 / 22261 | -38.05% (mostly IEAs: 8392)
*  NCBITaxon:4896\|Schizosaccharomyces pombe | -2712 / 52187 | -5.20% (mostly ISOs; -2248)
* NCBITaxon:36329\|Plasmodium falciparum 3D7 | -3727 / 18734 | -19.89% - IEAs out of date
* NCBITaxon:3702\|Arabidopsis thaliana | 7658 / 273816 | 2.80% - NDs; looks like they added missing ND, from looking at AT5G02630

Ensembl annotations: 
* NCBITaxon:9615\|Canis lupus familiaris | -6767 / 216273 | -3.13%
* NCBITaxon:9913\|Bos taurus | -22318 / 198500 | -11.24%
* NCBITaxon:9823\|Sus scrofa | 12944 / 222272 | 5.82%

**Comment by pgaudet** [comment]
Emailed Val about pombe and japonicus. 
> I am QC’ing the GO release, and we have lost quite a few pombe and japonicus annotations (IEAs and ISOs):
 ·  NCBITaxon:4897|Schizosaccharomyces japonicus | -8471 / 22261 | -38.05% (mostly IEAs: 8392)
·  NCBITaxon:4896|Schizosaccharomyces pombe | -2712 / 52187 | -5.20% (mostly ISOs; -2248)
Just wondering if that’s expected.
Unless this is a major problem for you, I’d like to go ahead with the release, since it’s been > 2 months since we had a release.

**Comment by pgaudet** [comment]
@ValWood 's reply

This is correct, it's because we changed the precedence now I have checked most IBA, so now we only submit IEA not covered by experimental or IBA.

(The problems were easier to spot when we gave IBA lower precedence, but we meant to make this change ages ago).

... at my end I only have 2080 IEA that are not already covered by experimental or IBA, but the difference is likely to be additional  IEA made by GO using F-P links.

v
